runtime-watcher/tests/e2e/utils/metrics.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"regexp"
+	"strconv"
+
+	"github.com/kyma-project/runtime-watcher/skr/internal/watchermetrics"
+)
+
+func ExposeSKRMetricsServiceEndpoint() error {
+	cmd := exec.Command("kubectl", "config", "use-context", "k3d-skr")
+	if _, err := cmd.CombinedOutput(); err != nil {
+		return fmt.Errorf("failed to switch context %w", err)
+	}
+
+	cmd = exec.Command("kubectl", "patch", "svc", "skr-webhook-metrics", "-p",
+		"{\"spec\": {\"type\": \"LoadBalancer\"}}", "-n", "kyma-system")
+	if _, err := cmd.CombinedOutput(); err != nil {
+		return fmt.Errorf("failed to patch metrics service %w", err)
+	}
+
+	return nil
+}
+
+func GetWatcherRequestDurationMetric(ctx context.Context) (float64, error) {
+	metricsBody, err := getMetricsBody(ctx)
+	if err != nil {
+		return 0, err
+	}
+
+	regex := regexp.MustCompile(`watcher_request_duration ([0-9]*\.?[0-9]+)`)
+
+	match := regex.FindStringSubmatch(metricsBody)
+	if len(match) < 1 {
+		return 0, fmt.Errorf("metric %s not found", watchermetrics.RequestDuration)
+	}
+
+	duration, err := strconv.ParseFloat(match[1], 64)
+	if err != nil {
+		return 0, fmt.Errorf("couldn't parse metric %s value", watchermetrics.RequestDuration)
+	}
+	return duration, nil
+}
+
+func GetKcpRequestsMetric(ctx context.Context) (int, error) {
+	metricsBody, err := getMetricsBody(ctx)
+	if err != nil {
+		return 0, err
+	}
+	regex := regexp.MustCompile(watchermetrics.KcpRequestsTotal + ` (\d+)`)
```

Comment:
In line 36 you named the metrics as a string instead of importing it from the metrics package. You should stay consistent, means either import them always or always write them as a string (as in line 36).

I would prefer to write them as string instead of importing, as in the KLM repo, since then if we change the name of the metrics we also need to change the test - Which is a wanted behaviour in terms of good testing style. (`Never import const into tests`)